hudi-common/src/main/java/org/apache/hudi/common/table/view/RemoteHoodieTableFileSystemView.java

```diff
@@ -165,17 +174,16 @@ private <T> T executeRequest(String requestPath, Map<String, String> queryParame
 
     String url = builder.toString();
     LOG.info("Sending request : (" + url + ")");
-    Response response;
     int timeout = this.timeoutSecs * 1000; // msec
-    switch (method) {
-      case GET:
-        response = Request.Get(url).connectTimeout(timeout).socketTimeout(timeout).execute();
-        break;
-      case POST:
-      default:
-        response = Request.Post(url).connectTimeout(timeout).socketTimeout(timeout).execute();
-        break;
-    }
+    Response response = (Response) retryHelper.tryWith(() -> {
+      switch (method) {
+        case GET:
+          return Request.Get(url).connectTimeout(timeout).socketTimeout(timeout).execute();
+        case POST:
+        default:
+          return Request.Post(url).connectTimeout(timeout).socketTimeout(timeout).execute();
+      }
+    }).start();
```

Comment:
Can we by default disable this feature ? You can add a new option similar like `hoodie.filesystem.operation.retry.enable`